temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +292,83 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
+     * that require additional customization may use {@link #setSslContext(SslContext)} directly.
+     *
+     * @param keyCertChainInputStream - an input stream for an X.509 certificate chain in PEM
+     *     format.
+     * @param keyPassword - the password of the keyFile, or null if it's not password-protected.
+     * @param keyInputStream - an input stream for a PKCS#8 private key in PEM format.
+     * @param fingerprints - a list of SHA1 fingerprints in hexadecimal form.
+     * @throws SSLException - when it was unable to build the context.
+     */
+    public Builder setCloudSslContext(
+        InputStream keyCertChainInputStream,
+        String keyPassword,
```

Comment:
the SslContextBuilder order is: cert chain, key, key password. Our helper methods seem to saw the keyPassword and KeyInput from what the ContextBuilder does. Can we have all the helper methods have the keyPassword after the keyInput/file for consistency?

Reply:
No, otherwise we'll be dealing with ambiguous signatures, (IS, IS, String, String..), in which case user would need to pass a string array as the last parameter. In my opinion it's better to add password in this position than having to deal with that issue.